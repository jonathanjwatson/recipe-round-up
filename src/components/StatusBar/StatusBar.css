.site-title {
    color: white;
}
.nav-item {
    color: white;
    padding-left: 1em;
}