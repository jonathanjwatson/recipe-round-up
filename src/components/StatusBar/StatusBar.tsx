import React from "react";
import "./StatusBar.css";

const StatusBar: React.FC = () => {
  return (
    <nav className="navbar navbar-expand-lg navbar-dark bg-dark">
      <h1 className="site-title">Recipe Round Up</h1>
      <button
        className="navbar-toggler"
        type="button"
        data-toggle="collapse"
        data-target="#navbarText"
        aria-controls="navbarText"
        aria-expanded="false"
        aria-label="Toggle navigation"
      >
        <span className="navbar-toggler-icon"></span>
      </button>
      <div className="collapse navbar-collapse" id="navbarText">
        <ul className="navbar-nav mr-auto">
          <li className="nav-item">Cancelled: 0</li>
          <li className="nav-item">Pending: 0</li>
          <li className="nav-item">In Progress: 0</li>
          <li className="nav-item">Fulfilled: 0</li>
        </ul>
      </div>
    </nav>
  );
};

export default StatusBar;
