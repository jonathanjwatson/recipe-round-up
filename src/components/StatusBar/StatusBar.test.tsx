import React from "react";
import { render, cleanup } from '@testing-library/react';
import StatusBar from "./StatusBar";

afterEach(cleanup);

it("renders", () => {
  const { asFragment } = render(<StatusBar />);
  expect(asFragment()).toMatchSnapshot();
});
