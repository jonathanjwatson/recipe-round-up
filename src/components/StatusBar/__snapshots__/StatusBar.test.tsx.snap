// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <nav
    class="navbar navbar-expand-lg navbar-dark bg-dark"
  >
    <h1
      class="site-title"
    >
      Recipe Round Up
    </h1>
    <button
      aria-controls="navbarText"
      aria-expanded="false"
      aria-label="Toggle navigation"
      class="navbar-toggler"
      data-target="#navbarText"
      data-toggle="collapse"
      type="button"
    >
      <span
        class="navbar-toggler-icon"
      />
    </button>
    <div
      class="collapse navbar-collapse"
      id="navbarText"
    >
      <ul
        class="navbar-nav mr-auto"
      >
        <li
          class="nav-item"
        >
          Cancelled: 0
        </li>
        <li
          class="nav-item"
        >
          Pending: 0
        </li>
        <li
          class="nav-item"
        >
          In Progress: 0
        </li>
        <li
          class="nav-item"
        >
          Fulfilled: 0
        </li>
      </ul>
    </div>
  </nav>
</DocumentFragment>
`;
